decision/openapi-3.yaml

```diff
@@ -534,6 +534,14 @@ components:
         adQuery:
           type: object
           nullable: true
+        floorPrice:
+          type: integer
```

Comment:
`type: integer` and `format: float` aren't compatible together. It either has to be `type: number` or `format: int32` or `format: int64`.

Reply:
Updated